### DISCUSSION TITLE
[Solved] Monitor new images and delete "older" images

### DISCUSSION THREAD

**Discussion by steffenweb** [containrrr/watchtower#1040] [General] [answered]
Hello i use watchtower with this configuration: 
```
docker run -d \
    --name watchtower --restart unless-stopped \
    -v /var/run/docker.sock:/var/run/docker.sock \
    -v /etc/timezone:/etc/timezone:ro \
    -e WATCHTOWER_NOTIFICATIONS=gotify \
    -e WATCHTOWER_NOTIFICATION_GOTIFY_URL="http://192.168.178.104:11230" \
    -e WATCHTOWER_NOTIFICATION_GOTIFY_TOKEN="AAAAAAAAA" \
    -e WATCHTOWER_MONITOR_ONLY=true \
    -e WATCHTOWER_CLEANUP=true \
    containrrr/watchtower  --schedule "* 0 15 * * *"
```

So watchtower pulls new Images and notify when there be a new one. My problem is now when i do not update my containers and another new image is available watchtower pulls the new and the older one will be still in the images so my drive will be filling with unused images. Any ideas to solve the problem? 

Thanks for help.

**Comment by piksel** [comment, marked as answer]
You can put [docker image prune](https://docs.docker.com/engine/reference/commandline/image_prune/) in crontab to run it daily/weekly or how often you want it to run. Or just run it manually.

**Reply by steffenweb** [reply]
I found this solution not very useful but i found my answer. I use diun to monitor the updates.

**Reply by piksel** [reply]
Sure! If you just want to be notified and not actually use the images, that seems like a much more purposeful solution. I have not used it though.
